documentation/modules/exploit/windows/misc/cisco_webex_ext.md

```diff
@@ -0,0 +1,52 @@
+Cisco WebEx is a suite of applications for online meeting organization and video conferencing.
+Included in this suite are extensions for popular web browsers which ease use and provide supplemental
+features.
+
+Version 1.0.1 of the WebEx extension for Google Chrome contains a vulnerability which allows an
+attacker to execute arbitrary commands on a target, which can lead to arbitrary remote code execution.
```

Comment:
The vulnerability doesn't allow for executing arbitrary code without some work.  If the context is some vulnerable gopher server in a ridiculous jailed environment where the only commands available are ping, fortune, ls, and sed, I could arbitrarily execute any of those, and still not have arbitrary code execution.

That's an arbitrary example, no pun intended, but more than a few organizations I've worked with make this distinction when categorizing exploit outcomes.